below/view/src/cgroup_tabs.rs

```diff
@@ -347,6 +350,12 @@ pub mod default_tabs {
         ])
     });
 
+    pub static CGROUP_PIDS_TAB: Lazy<CgroupTab> = Lazy::new(|| {
```

Comment:
Can we avoid making a separate PIDs tab and just include pids.current in General tab?

Reply:
Thanks, fixed.